rails/deprecated.md

```diff
@@ -0,0 +1,19 @@
+This image is officially deprecated in favor of [the standard `ruby` image](https://hub.docker.com/_/ruby/), and will receive no further updates. Please adjust your usage accordingly.
```

Comment:
Maybe we should give a leniency period rather than this abrupt end. How about an EOL of maybe 6 months from now?